frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -127,6 +127,21 @@ function ReaderHighlight:genHighlightDrawerMenu()
             end,
             callback = function()
                 self.view.highlight.disabled = not self.view.highlight.disabled
+                self.view.highlight.zoom =  self.view.highlight.disabled
```

Comment:
Dunno if we people should be allowed to disable both ? (for lazy people that let their fingers rest on the screen :)

Reply:
I Dunno ..so I will leave it as it is.